app/src/main/java/com/example/mywatchface/mvp/view/WatchFaceView.kt

```diff
@@ -0,0 +1,222 @@
+        canvas.drawLine(
+            mCenterX,
+            mCenterY - CENTER_GAP_AND_CIRCLE_RADIUS,
+            mCenterX,
+            mCenterY - sMinuteHandLength,
+            mMinutePaint
+        )
+        if (!mAmbient) {
+            canvas.rotate(secondsRotation - minutesRotation, mCenterX, mCenterY)
+            canvas.drawLine(
+                mCenterX,
+                mCenterY - CENTER_GAP_AND_CIRCLE_RADIUS,
+                mCenterX,
+                mCenterY - mSecondHandLength,
+                mSecondPaint
+            )
+
+        }
+        canvas.drawCircle(mCenterX, mCenterY, CENTER_GAP_AND_CIRCLE_RADIUS, mTickAndCirclePaint)
+        canvas.restore()
+    }
+
+    private fun tickAndCirclePaint() = Paint().apply {
+        color = mWatchHandColor
+        strokeWidth = SECOND_TICK_STROKE_WIDTH
+        isAntiAlias = true
+        style = Paint.Style.STROKE
+        setShadowLayer(
+            SHADOW_RADIUS, ZERO_FLOAT, ZERO_FLOAT, mWatchHandShadowColor
+        )
+    }
+
+    private fun secondsPaint() = Paint().apply {
+        color = mWatchHandHighlightColor
+        strokeWidth = SECOND_TICK_STROKE_WIDTH
+        isAntiAlias = true
+        strokeCap = Paint.Cap.ROUND
+        setShadowLayer(
+            SHADOW_RADIUS, ZERO_FLOAT, ZERO_FLOAT, mWatchHandShadowColor
+        )
+    }
+
+    private fun minutesPaint() = Paint().apply {
+        color = mWatchHandColor
+        strokeWidth = MINUTE_STROKE_WIDTH
+        isAntiAlias = true
+        strokeCap = Paint.Cap.ROUND
+        setShadowLayer(
+            SHADOW_RADIUS, ZERO_FLOAT, ZERO_FLOAT, mWatchHandShadowColor
+        )
+    }
+
+    private fun hoursPaint() = Paint().apply {
+        color = mWatchHandColor
+        strokeWidth = HOUR_STROKE_WIDTH
+        isAntiAlias = true
+        strokeCap = Paint.Cap.ROUND
+        setShadowLayer(
+            SHADOW_RADIUS, ZERO_FLOAT, ZERO_FLOAT, mWatchHandShadowColor
+        )
+    }
+
+    private fun updateInAmbientMode() {
+        mHourPaint.color = Color.WHITE
+        mMinutePaint.color = Color.WHITE
+        mSecondPaint.color = Color.WHITE
+        mTickAndCirclePaint.color = Color.WHITE
+        mHourPaint.isAntiAlias = false
+        mMinutePaint.isAntiAlias = false
+        mSecondPaint.isAntiAlias = false
+        mTickAndCirclePaint.isAntiAlias = false
+        mHourPaint.clearShadowLayer()
+        mMinutePaint.clearShadowLayer()
+        mSecondPaint.clearShadowLayer()
+        mTickAndCirclePaint.clearShadowLayer()
+    }
+
+    private fun updateInNormalMode() {
+        mHourPaint.color = mWatchHandColor
+        mMinutePaint.color = mWatchHandColor
+        mSecondPaint.color = mWatchHandHighlightColor
+        mTickAndCirclePaint.color = mWatchHandColor
+        mHourPaint.isAntiAlias = true
+        mMinutePaint.isAntiAlias = true
+        mSecondPaint.isAntiAlias = true
+        mTickAndCirclePaint.isAntiAlias = true
```

Comment:
this const true can be changed to ACTIVE for a more descriptive name

Reply:
done